testing/utilities.py

```diff
@@ -235,46 +322,40 @@ def test_get_inputs(self):
         '''
         
         inputs = requests.get('{0}/inputs'.format(self.url)).json()
-        self.assertEqual(len(inputs), len(self.inputs_ref))
-        for inp in inputs:
-            self.assertTrue(inp in self.inputs_ref)
-            self.assertTrue(inputs[inp]['Unit'] == self.inputs_ref[inp]['Unit'])
-            self.assertTrue(inputs[inp]['Description'] == self.inputs_ref[inp]['Description'])
-            self.assertTrue(inputs[inp]['Minimum'] == self.inputs_ref[inp]['Minimum'])
-            self.assertTrue(inputs[inp]['Maximum'] == self.inputs_ref[inp]['Maximum'])
+        ref_filepath = os.path.join(get_root_path(), 'testing', 'references', self.name, 'get_inputs.txt')
+        self.compare_ref_json(inputs, ref_filepath)
 
     def test_get_measurements(self):
         '''Test getting the measurement list of test.
         
         '''
 
         measurements = requests.get('{0}/measurements'.format(self.url)).json()
-        self.assertEqual(len(measurements), len(self.measurements_ref))
-        for measurement in measurements:
-            self.assertTrue(measurement in self.measurements_ref)
-            self.assertTrue(measurements[measurement]['Unit'] == self.measurements_ref[measurement]['Unit'])
-            self.assertTrue(measurements[measurement]['Description'] == self.measurements_ref[measurement]['Description'])
-            self.assertTrue(measurements[measurement]['Minimum'] == self.measurements_ref[measurement]['Minimum'])
-            self.assertTrue(measurements[measurement]['Maximum'] == self.measurements_ref[measurement]['Maximum'])
+        ref_filepath = os.path.join(get_root_path(), 'testing', 'references', self.name, 'get_measurements.txt')
```

Comment:
Same here, I think these reference files are missing.

Reply:
Same issue as previous comment.  Fixed in https://github.com/ibpsa/project1-boptest/pull/189/commits/b76cfb448de10d6b64a91e7f497c6de96709156c.